src/universal/auth/pages/LoginPage/LoginPage.tsx

```diff
@@ -55,7 +56,8 @@ export default class LoginPage extends Component<any, any> {
         <LoginForm
           isLoading={isLoggingIn}
           errorMessage={loginError}
-          onSubmit={data => this.handleSubmit(data)}/>
```

Comment:
tslint is complaining about lambdas in components, it says it will result in performance problems.